src/common/policy.c

```diff
@@ -0,0 +1,128 @@
+/**
+ * (C) Copyright 2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * This file is part of daos. It implements some miscellaneous policy functions
+ */
+
+#include "policy.h"
+#include <string.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#define POLICY_BAD_PARAM 	(-1)
+
+static const char *POLICY_NAMES[DAOS_MEDIA_POLICY_MAX] =
+{"io_size",
+"write_intensivity"};
+
+static const char *POLICY_PARAMS[DAOS_MEDIA_POLICY_MAX]
+				[DAOS_MEDIA_POLICY_PARAMS_MAX] =
+{
+{"th1","th2","",""},				/* io_size */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)